packs/world-altar/data/world-altar/functions/sacrifice/main.mcfunction

```diff
@@ -5,6 +5,9 @@ execute if block ~ ~2 ~ minecraft:netherite_block run function world-altar:sacri
 # Handle special block sacrifices
 execute if block ~ ~2 ~ minecraft:lapis_block run function world-altar:sacrifice/lapis
 
+# Handle invalid sacrifices
+execute unless block ~ ~2 ~ minecraft:air run function world-altar:sacrifice/invalid
```

Comment:
i'm making an assumption here that for a valid sacrifice, the previous functions would have run and replaced the block with air, is that true?